scripts/bash/enceladus_env.template.sh

```diff
@@ -46,8 +57,20 @@ CONF_DEFAULT_DRIVER_MEMORY=""
 CONF_DEFAULT_DRIVER_CORES=""
 CONF_DEFAULT_EXECUTOR_MEMORY=""
 CONF_DEFAULT_EXECUTOR_CORES=""
+CONF_DEFAULT_EXECUTOR_CORES=""
+# setting num executors disables DRA
 CONF_DEFAULT_NUM_EXECUTORS=""
 
+# Dynamic Resource Allocation
+# also enables external shuffle service and adaptive execution for consistent setup
+CONF_DEFAULT_DRA_ENABLED=false
+# max executors limit is a required parameter
+CONF_DEFAULT_DRA_MAX_EXECUTORS=4
+
+CONF_DEFAULT_DRA_MIN_EXECUTORS=0
+CONF_DEFAULT_DRA_ALLOCATION_RATIO=0.5
+CONF_DEFAULT_ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE=134217728
```

Comment:
This number seems little random. Can you add a comment please, why this number? (128MB?)

Reply:
It is 1 HDF block by default. The idea is to match Spark partitions to HDFS blocks. It would allow less load to name nodes and potentially reduce amount of repartitioning happening in Spark.

Reply:
Oh, I see. Would be a nice educative comment in the script 😉